lib/std/c.zig

```diff
@@ -276,6 +276,7 @@ pub extern "c" fn pthread_key_create(key: *pthread_key_t, destructor: ?fn (value
 pub extern "c" fn pthread_key_delete(key: pthread_key_t) c_int;
 pub extern "c" fn pthread_getspecific(key: pthread_key_t) ?*c_void;
 pub extern "c" fn pthread_setspecific(key: pthread_key_t, value: ?*c_void) c_int;
+pub extern "c" fn pthread_getname_np(thread: std.c.pthread_t, name: [*:0]const u8, len: usize) c_int;
```

Comment:
Non portable definitions go into their own `<platform>/c.zig` file, this is not available when targeting eg. Windows.

Reply:
ok, I'll move it.